frontend/src/static/js/v_summary.js

```diff
@@ -761,6 +761,10 @@ window.vSummary = {
       return filtered;
     },
 
+    getPercentile(index) {
+      return (Math.round(index * 1000 / (this.filtered.length - 1)) / 10).toFixed(1);
```

Comment:
Is there a particular reason for using this way of computing instead of 
`((index * 100) / (this.filtered.length - 1)).toFixed(1)`?

Reply:
According to my search `ToFixed` might get different result on different browsers, depending on how they implemented it. The `math.round` was used to ensure the consistency of the result calculated across browsers.